### PR TITLE
kic: Increase inspect timeout to 10s

### LINKED ISSUE #7193: Docker flake: Unable to get machine status: state: inspect <profile> timeout
as seen here https://storage.googleapis.com/minikube-builds/logs/7173/67bd258/Docker_Linux.html#fail_TestFunctional%2fparallel%2fUpdateContextCmd

### PR DESCRIPTION
**Pull request by tstromberg**
Heavy docker loads can sometimes slow this down.

Fixes #7193

### CODE REVIEW

File: pkg/drivers/kic/kic.go [hunk 1]
```diff
@@ -220,8 +220,7 @@ func (d *Driver) GetURL() (string, error) {
 
 // GetState returns the state that the host is in (running, stopped, etc)
 func (d *Driver) GetState() (state.State, error) {
-	// allow no more than 2 seconds for this. when this takes long this means deadline passed
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 8*time.Second)
```
Comment (medyagh) [inline pkg/drivers/kic/kic.go]: any chance we could still warn the user to restart their docker, when it takes that long ?
in 90% of the times if inspect takes too much time, restarting docker makes it super fast.

Comment (tstromberg) [reply pkg/drivers/kic/kic.go]: Will add a warning.

File: pkg/drivers/kic/kic.go [hunk 2]
```diff
@@ -220,16 +220,21 @@ func (d *Driver) GetURL() (string, error) {
...
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
-
+	
+	start := time.Now()
 	cmd := exec.CommandContext(ctx, d.NodeConfig.OCIBinary, "inspect", "-f", "{{.State.Status}}", d.MachineName)
 	out, err := cmd.CombinedOutput()
+	
+	if time.Since(start) > 2*time.Second {
+		glog.Errorf("%s took an unusually long time. Restarting the %s daemon may improve performance.", cmd, d.OCIBinary)
```
Comment (medyagh) [inline pkg/drivers/kic/kic.go]: did u mean cmd.Args ? 
also quote the command %q

**Comment by medyagh** [comment]
do-not-merge/invalid-commit-message ?

**Comment by tstromberg** [comment]
I'm going to re-open this from my fork.

**Pull request closed without merging**